apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -64,8 +64,6 @@ const FilterBar = ({
     return count
   }
 
-  useEffect(() => { searchRef.current && searchRef.current.focus()})
```

Comment:
I originally attempted to solve this with `autoFocus` but as you say it doesn't support it. I'll add the second argument, maybe it will fix the first problem regarding the initial load of the large layout automatically focusing on TextInput